integrated_channels/blackboard/migrations/0010_auto_20211221_1532.py

```diff
@@ -0,0 +1,37 @@
+# Generated by Django 3.2.10 on 2021-12-21 15:32
+
+from django.db import migrations, models
+import uuid
+
+
+def create_uuid(apps, schema_editor):
```

Comment:
do we need a reverse data migration to undo this for rollbacks?

Reply:
we can throw them away.